File: pinyougou-sellergoods-interface/src/main/java/com/pinyougou/sellergoods/service/OrderService.java
package com.pinyougou.sellergoods.service;

import java.util.List;

import com.pinyougou.pojo.TbOrder;
import com.pinyougou.pojogroup.WorkBook;

import entity.PageResult;

public interface OrderService {

	
	public List<TbOrder> findOrderList(WorkBook workBook);
	
	PageResult findOrderList(String status,String sellerId,int pageNum,int pageSize);
	
	void updateStatus(String status,Long orderId);
}

File: pinyougou-shop-web/src/main/webapp/js/service/orderService.js
app.service("orderService",function($http){
	
	this.exportXls=function(status,date1,date2){
		
		return $http.get('../order/exportXls.do?status='+status+'&date1='+date1+'&date2='+date2);
		
	}
	
	this.findOrder=function(status,pageNum,pageSize){
		
		return $http.get('../order/findOrder.do?status='+status+'&pageNum='+pageNum+'&pageSize='+pageSize);
		
	}
	this.updateStatus=function(status,orderId){
		
		return $http.get('../order/updateStatus.do?status='+status+'&orderId='+orderId);
		
	}
	
})